internal/runner/inputs.go

```diff
@@ -31,11 +30,6 @@ func (r *Runner) initializeTemplatesHTTPInput() (*hybrid.HybridMap, error) {
 	}
 	gologger.Info().Msgf("Running httpx on input host")
 
-	var bulkSize = probeBulkSize
-	if r.options.BulkSize > probeBulkSize {
-		bulkSize = r.options.BulkSize
-	}
-
```

Comment:
i think this should be allowed , while the goal of adaptive speed is to effectively tune and control , initalizeTemplateHTTPInput acts as preflight before actually starting vulnerability scan , 

this will be a bad experience if i have given say 40k targets in input , since concurrency will always be <= 50 it would take forever to just complete probing , arguably one should run http on input before passing targets to nuclei 🤷 

maybe a seperate option to control probe concurrency would be good idea 🤔

Reply:
Yeah, but on the other side it's almost always the used value, since most of the runs happen with default value of 25. Probably we can add another option and initialize it within one of the various global inits (ex. `protocolstate`)